### PR TITLE
#41 [part 1] Bot API 8.0+: Added multiple fields to WebApp

### PR DESCRIPTION
**Pull request by OlegNovosad**
#41 [part 1]

Fields added:

`isActive`
`isFullscreen`
`isOrientationLocked`
`safeAreaInset`
`contentSafeAreaInset`

### CODE REVIEW

**Comment by OlegNovosad** [comment]
@khamidjon-khamidov I have a question on testing an example app of this one. How do you usually test it without using fake telegram web app instance?

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (khamidjon-khamidov) [inline .vscode/launch.json:1]: @OlegNovosad could you add this file to .gitignore

**Comment by khamidjon-khamidov** [comment]
> @khamidjon-khamidov I have a question on testing an example app of this one. How do you usually test it without using fake telegram web app instance?

I have a mini test app freely hosted on Firebase: http://t.me/bozorbop_bot/bozorchi . For the library, I release my changes there and test it.

**Comment by khamidjon-khamidov** [comment]
@OlegNovosad could you also mention on which apis you are working. I will work on other api methods

File: .vscode/launch.json (hunk 1 above)
Comment (OlegNovosad) [reply .vscode/launch.json:1]: will add in next PR

**Comment by OlegNovosad** [comment]
@khamidjon-khamidov I will be working on full-screen and location manager mainly as for now.

**Pull request merged**